docs/learn/submit-data.md

```diff
@@ -58,6 +58,50 @@ The `GasCostPerBlobByte` and `GasCostPerTransactionByte` are parameters that
 could potentially be adjusted through the system's governance mechanisms. Hence,
 actual costs may vary depending on the current state of these parameters.
 
+#### Gas fee calculation
+
+The total fee for a transaction is calculated as the product of the gas limit
+for the transaction and the gas price set by the user:
+
+$\text{Total Fee} = \text{Gas Limit} \times \text{Gas Price}$
+
+The gas limit for a transaction is the maximum amount of gas that a user is
+willing to spend on a transaction. It is determined by both a static
+"fixed cost" and a dynamic cost based on the size of each blob involved
+in the transaction:
+
+$\text{Gas Limit} = \text{Fixed Cost} + \sum_{i=1}^{n} \text{SparseSharesNeeded(Blob}_i) \times \text{Share Size} \times \text{Gas Cost Per Blob Byte}$
+
+Where:
+
+- $\text{Fixed Cost}$ is a static value (default is 65,000 gas)
+- $\text{SparseSharesNeeded(Blob}_i)$ is the number of shares needed for the $i$th blob in the transaction
```

Comment:
The `$i$` here doesn't show up as math on GitHub. Will it in the docs renderer?

Reply:
<img width="1451" alt="scrot 2023-09-18 at 7 55 42 PM" src="https://github.com/celestiaorg/docs/assets/46639943/07601557-c5f7-431e-9e8a-a23dcc219fed">
it looks good to me @adlerjohn

Reply:
https://docs.celestia.org/pr-preview/pr-1051/learn/submit-data/#gas-fee-calculation